src/infrastructure/services/GoogleAuthService.ts

```diff
@@ -82,4 +86,15 @@ export class GoogleAuthService {
       })
       .then(response => response.data)
   }
+
```

Comment:
I think the `GoogleAuthService` should implement a `Authservice` interface so that we can migrate to any identity provider in the future.